log4j-api/src/main/java/org/apache/logging/log4j/util/PropertiesUtil.java

```diff
@@ -573,7 +573,7 @@ public boolean isOsWindows() {
     private enum TimeUnit {
         NANOS("ns,nano,nanos,nanosecond,nanoseconds", ChronoUnit.NANOS),
         MICROS("us,micro,micros,microsecond,microseconds", ChronoUnit.MICROS),
-        MILLIS("ms,milli,millis,millsecond,milliseconds", ChronoUnit.MILLIS),
+        MILLIS("ms,milli,millis,millisecond,milliseconds", ChronoUnit.MILLIS),
```

Comment:
This might be an api change, but it probably means that `millisecond` doesn't work